Backend/flaskProject/main.py

```diff
@@ -170,75 +166,6 @@ def burndown_graph():
     return render_template("burndown-graph.html")
 
 
-# TODO: Depricated (Endpoint not in use)
-@app.route("/burndown-metric-parameter", methods=["GET", "POST"])
-def burndown_metric_parameter():
-    if "auth_token" not in session:
-        return redirect("/")
-
-    if request.method == "POST":
-        parameter = request.form.get("workOption")
-        if parameter == "businessValues":
-            return redirect("/burndown-bv")
-        return redirect("/burndown-metric_configuration")  # just a placeholder
-
-    return render_template("burndown-metric_configuration.html")
-
-
-# TODO: Depricated (Endpoint not in use)
-@app.route("/work-done-chart", methods=["GET"])
-def work_done_chart():
-    if "auth_token" not in session:
-        return redirect("/")
-
-    auth_token = session["auth_token"]
-    taiga_url = os.getenv("TAIGA_URL")
-
-    print(request.args)
-
-    project_id = request.args.get("projectid")
-    sprint_id = request.args.get("sprintid")
-
-    if (not project_id) or (not sprint_id):
-        return "Invalid request!"
-
-    milestones_api_url = f"{taiga_url}/milestones/{sprint_id}?project={project_id}"
-    taks_api_url = f"{taiga_url}/tasks?project={project_id}"
-
-    # Define headers including the authorization token and content type
-    headers = {
-        "Authorization": f"Bearer {auth_token}",
-        "Content-Type": "application/json",
-    }
-
-    try:
-        # Make a GET request to Taiga API to retrieve user stories
-        mailstones_res = requests.get(milestones_api_url, headers=headers)
-        mailstones_res.raise_for_status()  # Raise an exception for HTTP errors (4xx or 5xx)
-
-        # Extracting information from the mailstones_res
-        milestone = mailstones_res.json()
-
-        # Make a GET request to Taiga API to retrieve tasks
-        tasks_res = requests.get(taks_api_url, headers=headers)
-        tasks_res.raise_for_status()  # Raise an exception for HTTP errors (4xx or 5xx)
-
-        # Extracting information from the tasks_res
-        tasks = tasks_res.json()
-
-        for user_story in milestone["user_stories"]:
-            if user_story["total_points"] == None:
-                continue
-
-        print(json.dumps({"milestone": milestone, "tasks": tasks}, indent=2))
-
-        return json.dumps(milestone)
-    except requests.exceptions.RequestException as e:
-        # Handle errors during the API request and print an error message
-        print(f"Error fetching project by slug: {e}")
-        return redirect("/error")
-
-
```

Comment:
This diff is already being removed in [PR-56](https://github.com/keer-asu/SER516-Team-Miami/pull/56/files#diff-f6f53975fb4c54131ba3b585b0800068dc6ce75cb1813db510a930de1444eb41L173-L241). Can be removed